### PR TITLE
Add issue for pulling on Mesos/Marathon

### PR DESCRIPTION
**Pull request by josephschorr**

### CODE REVIEW

File: issues/quay-mesos.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+---
+layout: doc
+sublayout: guide
+title: Pulling Private Quay.io images with Marathon/Mesos fails
```
Comment (jzelinskie) [inline issues/quay-mesos.md:4]: This is a guide not a solution

File: issues/quay-mesos.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+```
+msg="Error: Status 403 trying to pull repository repo/project: \"{\\\"error\\\": \\\"Permission Denied\\\"}\""
+```
+
+### Workaround
+
+To get Marathon/Mesos to pull from a private repository, we need to copy the `.docker/config.json`
+credentials onto the worker machines.
+
+### Step 1: Create or Download Credentials
```
Comment (jzelinskie) [inline issues/quay-mesos.md]: Steps should be `####`

**Comment by josephschorr** [comment]
Fixed

File: issues/quay-mesos.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+```
+"uris": [
+  "file:///etc/docker.tar.gz"
+]
+```
+
+For example:
+
+```
```
Comment (jzelinskie) [inline issues/quay-mesos.md]:
``````
```json
``````

Comment (josephschorr) [inline issues/quay-mesos.md]: done

**Comment by jzelinskie** [comment]
LGTM

**Pull request merged**